contract_compassion/tests/test_base_module.py

```diff
@@ -20,8 +20,8 @@ class test_base_module(test_base_contract):
 
     def setUp(self):
         # Retrieve of income account
-        account_obj = self.env['account.account']
-        property_account_income = account_obj.search([
+        super(test_base_module, self).setUp()

```

Comment:
Right before you told me you would not call super from here and now you add it. Please explain.